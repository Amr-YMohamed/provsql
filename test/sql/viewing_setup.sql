\set ECHO none
SET search_path TO public, provsql;

/* Add descriptions to the personnel table */
ALTER TABLE personnel ADD COLUMN desc_str VARCHAR(255);
UPDATE personnel SET desc_str='personnel.'||name;

SELECT create_provenance_mapping('d', 'personnel', 'desc_str');

SELECT value FROM d ORDER BY value;
