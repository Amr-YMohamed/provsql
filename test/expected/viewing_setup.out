\set ECHO none
 create_provenance_mapping 
---------------------------
 
(1 row)

       value       
-------------------
 personnel.Dave
 personnel.Ellen
 personnel.John
 personnel.Magdalen
 personnel.Nancy
 personnel.Paul
 personnel.Susan
(7 rows)

